### ISSUE TITLE
Additives inconsistencies between app and website

### ISSUE THREAD

**Issue by asmoth360** [openfoodfacts/openfoodfacts-androidapp#451]
I noticed the additives were now in red in the ingredients list, probably to match their "toxicity". Maybe the problem is not new but anyway now it says many additives are toxic, when on the website it's not.
Also, the additive E160aii  is not displayed at all in the app.

Example with the product 3103220033371 (https://fr.openfoodfacts.org/produit/3103220033371/tagada-l-originale-haribo) (sorry it's in french) :
Additive - Website dangerosity   - App dangerosity
E428      -               /                   -  Modérément toxique
E330      - peu ou pas dangereux - Modérément toxique
E100      - pas ou peu de risques - Modérément toxique
E120      - risques modérés         - Douteux
E160aii   -               /                   - Not displayed in the app

So basically everything in this example is in red and it seems I'm going to die if I eat it!

**Comment by teolemon** [comment]
Another non-off source is used in the app. #149

**Comment by asmoth360** [comment]
Wouldn't it be possible to use the same source?

**Comment by teolemon** [comment]
fixed. closing.
